src/osemgrep/targeting/Gitignore_level.ml

```diff
@@ -0,0 +1,9 @@
+(*
+   A level as defined in the gitignore specification.
+*)
+
+type t = {
```

Comment:
when the .ml mostly contain types, I usually don't define a corresponding .mli since it's mostly a copy-paste.
I would just move all the good comments in Gitignore_level.ml and get rid of Gitignore_level.mli

Reply:
I agree - didn't realize at first it would be only a type definition.